docs/release_checklist.md

```diff
@@ -149,24 +124,24 @@ git push upstream $RELEASE_CANDIDATE_NAME
 
 CircleCI will automatically create a tagged release image and client binary to test with.
 
-For testers, the process to start testing after CircleCI finishes building the artifacts involves the following steps to grab the client from Google Cloud Storage:
+After CircleCI finishes building the artifacts, the use the following commands to fetch the client for testing:
```

Comment:
small nit here if you want to change it: :s/the use/use

Reply:
fixed. Thanks again